azure-pipelines.yml

```diff
@@ -67,8 +62,13 @@ jobs:
       - linux: py38-online
         name: py38_test_online
 
-      - linux32: py38-32bit
+      - manylinux: py38-32bit
         name: py38_test_32bit
+        manylinux_image: manylinux1_i686
```

Comment:
Can we not use an update to date image?

Reply:
I think more of our deps provide wheels for this, which should make it a lot faster, but we should double check that.